tools/init_dev.js

```diff
@@ -64,7 +65,17 @@ if (process.argv.length > 2 && process.argv[2] === '-f') {
   force = true;
 }
 
-const futures = [];
+// need to manually delete ts under midway since midway does not handle yarn installation properly
+const removeMidwayTS = () => new Promise((resolve, reject) => {
+  const tsPath = path.join(__dirname, "../", "./node_modules/midway-bin/node_modules/typescript/");
```

Comment:
Shall we have a discussion about this with midway team?

Reply:
And we should use single quotes with string.